### PR TITLE
second release to production

### PR DESCRIPTION
**Pull request by savaukr**

### CODE REVIEW

**Review by savaukr — COMMENTED** [review]
ok

**Review by savaukr — COMMENTED** [review]
ok

**Comment by savabinary** [comment]
я не знаю як пslndthlbnb

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# resume
+# Resume (edited)
```
Comment (savabinary) [inline README.md:1]: ok

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (savabinary) [inline index.html:1]: ok

**Review by savabinary — COMMENTED** [review]
the end

**Comment by savaukr** [comment]
ok

**Review by savabinary — COMMENTED** [review]
I am reviewed

**Review by savabinary — COMMENTED** [review]
I am reviewed

**Review by savabinary — CHANGES_REQUESTED** [review]
Finally, i understanded, how it works

**Review by savabinary — CHANGES_REQUESTED** [review]
All right

**Review by savabinary — APPROVED** [review]

**Pull request merged**